src/crypto/tls/common.go

```diff
@@ -673,7 +678,20 @@ type Config struct {
 	// used for debugging.
 	KeyLogWriter io.Writer
 
-	// mutex protects sessionTicketKeys and autoSessionTicketKeys.
+	// SupportDelegatedCredential is true if the client or server is willing
+	// to negotiate the delegated credential extension.
+	// This can only be used for TLS 1.3
+	//
+	// See https://tools.ietf.org/html/draft-ietf-tls-subcerts.
+	SupportDelegatedCredential bool
+
+	// GetDelegatedCredential returns a DelegatedCredential for use with the
+	// delegated credential extension based on the ClientHello and TLS version
+	// selected for the session. If this is nil, then the server will not offer
+	// a DelegatedCredential.
+	GetDelegatedCredential func(*ClientHelloInfo) (*DelegatedCredential, crypto.PrivateKey, error)
+
+        // mutex protects sessionTicketKeys and autoSessionTicketKeys.
```

Comment:
Nit: spacing.

Reply:
It seems like this is just git rendering it that way, as locally there is no space.

Reply:
Yes.. this seems like git is incorrectly rendering it..